config/locales/en.yml

```diff
@@ -54,16 +54,28 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    admin_hidden_work_notification:
+      subject: "[%{app_name}] Your work has been hidden by the Abuse Team"
+      hello: "Dear %{user},"
+      tos: "Terms of Service"
+      abuse: "contact our Abuse Team"
+      sincerely: "Sincerely,"
+      ao3_abuse: "AO3 Abuse"
+      part2: "We are investigating the matter and will contact you once the investigation has been completed. While the work is under investigation it has been hidden; only you will be able to access it."
+      html:
+        part1: "Your work <i>%{title}</i> has been reported as being in violation of the Archive of Our Own's %{tos}."
+        part3: "If you have any questions, please contact our Abuse Team (%{abuse})"
```

Comment:
Oh, also! There should be a period at the end of the sentence for part3 of the HTML email, i.e. `part3: "If you have any questions, please %{abuse}."` (The text email's better without it, though.)